src/main/java/Duke.java

```diff
@@ -1,10 +1,55 @@
+package src.main.java;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("----------------------------------------------");
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        System.out.println("----------------------------------------------");
+        String userInput;
+        ArrayList<Task> toDo = new ArrayList<>();
+        do {
+            Scanner in = new Scanner(System.in);
+            userInput = in.nextLine();
+            if (userInput.equalsIgnoreCase("list")){        //user checks list
+                System.out.println("----------------------------------------------");
+                System.out.println("Here are the tasks in your list");
+                for (int i=0; i<toDo.size(); i++){
+                    System.out.println(i+1 + ".[" + toDo.get(i).getStatusIcon() + "] " + toDo.get(i).toString());
```

Comment:
You can try move 'print' into a seperate function as the style will be changed in later levels.